js/script.js

```diff
@@ -0,0 +1,298 @@
+
+// Create keyboard buttons
+const createKeyboard = (container) => {
+    for (let i = 65; i <= 90; i++) {
+        const keyboardsBtn = document.createElement('button');
+        keyboardsBtn.type = 'button';
+        keyboardsBtn.classList.add('keyboards__button');
+        keyboardsBtn.setAttribute('data-code', `Key${String.fromCharCode(i)}`);
+        keyboardsBtn.innerText = String.fromCharCode(i); // Keyboard
+        container.appendChild(keyboardsBtn);
+    }
+}
+createKeyboard(keyboardContainer);
+
+// KeyA => a
+const formaKeyboardCode = (code) => code[code.length - 1].toLowerCase();
+
+
+// Get Random questions
+const getRandomQuestion = (question) => {
+    const { word, hint } = wordList[Math.floor(Math.random() * wordList.length)];
+    console.log('correct answer: ', word);
+    question.innerHTML = 'Hint: ' + `<b>${hint}</b>`;
+    createListItem(listWords, word.length);
+
+    return word;
+}
+const answer = getRandomQuestion(returnQuestion);
+
+//Open modal window
+const callPopUpWindow = () => {
+    const ulList = document.querySelector('.hangmanGame__list');
+
+    return Array.from(ulList.children).every((item) => item.innerHTML);
+}
+
+const handleClick = (event) => {
+    let letter;
+    const ul = document.querySelector('.hangmanGame__list');
+    const liItems = Array.from(ul.children);
+    const button = document.querySelector(`[data-code = ${event.code}]`);
+    
+    // Disabled click buttons
+    const arrBtn = Array.from(keyboardContainer.children);
+
+    for(let i = 0; i <= arrBtn.length; i++) {
+        if(arrBtn[i] === event.target || arrBtn[i] === button) {
+            arrBtn[i].setAttribute('disabled', 'disabled');
+            arrBtn[i].style = 'background-color: rgb(151, 141, 158)';
+        }
+    }
+
+    if(event.code) {
+        letter = formaKeyboardCode(event.code);
+
+    } else {
+        letter = event.target.innerHTML;
+    }
+
+    if (answer.includes(letter)) {
+        const answerArr = answer.split('');
+        answerArr.map((item, index) => {
+
+            if (item === letter) {
+
+                liItems[index].innerHTML = letter;
+                liItems[index].style = 'border-bottom: none';
+
+            }
+        })
+    } else if (count <= 6) {
+
+        returnAttempt.innerHTML = `Incorrect guesses: <span>${count} / 6</span>`;
+        const className = arrayHangmanBody[count].className;
+        document.querySelector(`.${className}`).style.display = 'block';
+        count++;
+
+    }
+
+    if(callPopUpWindow()) {
+        document.querySelector('.popUp_wrapper').style = 'visibility: visible';
+        Object.assign(document.querySelector('.main').style, {filter: 'blur(5px)', pointerEvents: 'none'});
+    }
+
+    if(count === (6 + 1)) {
+        document.querySelector('.modal__window_container').style = 'visibility: visible';
+        Object.assign(document.querySelector('.main').style, {filter: 'blur(5px)', pointerEvents: 'none'});
+    }
```

Comment:
```suggestion
   const isWin = checkIfAllItemsFilled();
    const isGameOver = count === 6;
  
    if(isWin) {
      document.querySelector('.popUp_wrapper').style = 'visibility: visible';
      Object.assign(document.querySelector('.main').style, { filter: 'blur(5px)', pointerEvents: 'none' });
      
      document.removeEventListener("keydown", handleClick);
  }

    if(isGameOver) {
      document.querySelector('.modal__window_container').style = 'visibility: visible';
      Object.assign(document.querySelector('.main').style, { filter: 'blur(5px)', pointerEvents: 'none' });
      
      document.removeEventListener("keydown", handleClick);
    }
```

во-первых: именнование переменных - крайне важно. по твоим условиям ничего непонятно, что происходит в коде. старайся уделять этому больше внимания

во-вторых: обрати внимание на удаление обработчика